src/main.py

```diff
@@ -88,6 +89,16 @@ def write_report(tests, drive, output_path):
 
     r.close()
 
+def restore_drive(config):
+    drive = config['drive']['name']
+    psid = config['drive']['psid']
+    logger.info(f"Cleaning up drive {drive} through a format.")
+
+    # Start with a PSID reset, just in case it's in a weird state
+    sedutil.reset_via_psid(drive, psid)
```

Comment:
Can we change this to only do this _if_ the PSID is set.  There could be a tolerance file where a PSID is not actually set.

Reply:
In case you're wondering - think SED but not OPAL/RUBY

Reply:
makes sense, I updated the code. Thanks.